src/diamond/handler/netuitive_handler.py

```diff
@@ -195,6 +195,25 @@ def get_default_config(self):
             'trim_backlog_multiplier': 4,
         })
         return config
+       
+       def get_default_config(self):
```

Comment:
Won't this function conflict with line 180? I don't see this picking up the new property you expect, either.

Reply:
else can i add only the below in same def

config = super(NetuitiveHandlerApp, self).get_default_config()

        config.update({
            'url': 'https://api.uat.netuitive.com/ingest/infrastructure',
            'api_key': 'apikey',
            'tags': None,
            'relations': None,
            'location': None,
            'batch': 100,
            'max_backlog_multiplier': 5,
            'trim_backlog_multiplier': 4,